tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/TupleAccessExprTest.java

```diff
@@ -201,6 +202,37 @@ public void testTupleAccessUsingUnionWithFiniteTypesNegative() {
         BRunUtil.invoke(compileResult, "testTupleAccessUsingUnionWithFiniteTypesNegative");
     }
 
+    @Test(dataProvider = "TupleAccessUsingCustomTypesFunctionList")
+    public void testTupleAccessUsingCustomTypes(String funcName) {
+        BRunUtil.invoke(compileResult, funcName);
+    }
+
+    @Test(dataProvider = "TupleWithRestTypesAccessFunctionList")
+    public void testTupleWithRestTypesAccess(String funcName) {
+        BRunUtil.invoke(compileResult, funcName);
+    }
+
+    @DataProvider(name = "TupleAccessUsingCustomTypesFunctionList")
+    public Object[][] getFunctionNames() {
```

Comment:
The name of the method could be reflective of the actual tests instead of specifying it as the name. See `org.ballerinalang.test.statements.matchstmt.MatchStmtListMatchPatternTest#dataToTestListMatchPatterns` for example.

Reply:
Updated. 1ef87477be4270f73cec77393ed6e541fc0886eb